src/index.js

```diff
@@ -1,19 +1,60 @@
-const addDebetInput = document.getElementById("add-debet-input");
-addDebetInput.addEventListener("click", addDebetLine);
-const addCreditInput = document.getElementById("add-credit-input");
-addCreditInput.addEventListener("click", addCreditLine);
+//this code delates inputs
+function deliteInput(fun) {
```

Comment:
What does `fun` argument mean? Is it function? As per code inside of function, it behaves like a DOM element.
BE EXTREMELY CAREFUL WITH NAMING AND TYPOS!!!